app/interfaces/api/v2/query_helper.rb

```diff
@@ -79,14 +69,24 @@ def unallocated_sql
             ON o.offence_class_id = oc.id
           LEFT OUTER JOIN transfer_details AS dt
             ON c.id = dt.claim_id
+          LEFT JOIN LATERAL (
+            SELECT CAST(error_messages AS VARCHAR), succeeded, deleted_at
+              FROM injection_attempts last_ia
+              WHERE last_ia.claim_id = c.id
+                AND last_ia.deleted_at is NULL
+              ORDER BY last_ia.created_at DESC
+              LIMIT 1
+            ) AS last_injection_attempt ON 1=1
         WHERE
           c.deleted_at IS NULL
-          AND c.type IN CLAIM_TYPES_FOR_SCHEME
+          AND c.type IN ('Claim::AdvocateClaim', 'Claim::AdvocateInterimClaim')
```

Comment:
I believe `CLAIM_TYPES_FOR_SCHEME` is needed for dynamic scheme context via the block below


```
     helpers do
          def in_statement_for(arr)
            arr.map(&:to_s).join('\', \'').prepend('(\'').concat('\')')
          end

          def claim_types_for_scheme
            return in_statement_for(::Claim::BaseClaim.agfs_claim_types) if scheme.eql?('agfs')
            in_statement_for(::Claim::BaseClaim.lgfs_claim_types)
          end

          def claims
            built_sql = unallocated_sql.gsub(/CLAIM_TYPES_FOR_SCHEME/, claim_types_for_scheme)
            result = ActiveRecord::Base.connection.execute(built_sql).to_a
            JSON.parse(result.to_json, object_class: OpenStruct)
          end

          def scheme
            params[:scheme]
          end
        end
```

to discuss, there should be a breaking test from this change....unless i am missing something

Reply:
Absolutely correct... this was a cut'n'paste error from getting the query updated :(